network/oauth_client.go

```diff
@@ -96,3 +101,16 @@ func (oc OAuthClient) RoundTrip(request *http.Request) (*http.Response, error) {
 
 	return client.Transport.RoundTrip(request)
 }
+
+func (oc OAuthClient) tokenRetry() (token *oauth2.Token, err error){
+	count := 0
+	for (count < retryCount) {
+		token, err = oc.oauthConfig.PasswordCredentialsToken(oc.context, oc.username, oc.password)
+		if e, ok := err.(net.Error); !(ok && e.Timeout()) {
+			return
+		}
+		count += 1
+		time.Sleep(30 * time.Second)
+	}
+	return
```

Comment:
Does https://github.com/pivotal-cf/om/pull/10 cover these changes? Are you interested in this PR as well?